Gemfile

```diff
@@ -40,6 +40,8 @@ group :development, :test do
   gem 'byebug', platform: :mri
   gem 'rspec-rails', '~> 3.5'
   gem 'shoulda-matchers'
+  gem 'pry'
```

Comment:
So we don't need these because we have byebug. 

I looked it up and found [this link](http://6ftdan.com/allyourdev/2015/02/20/some-basic-ruby-tools-for-sniffing-out-errors/)